### ISSUE TITLE
Real time on webcam?

### ISSUE THREAD

**Issue by klwong126** [eugenelet/Meta-rPPG#11]
Is the model capable of running real time on a webcam?

**Comment by eugenelet** [comment]
Yes, we have even tested it on Nvidia AGX Xavier and it is able to run above 30 FPS.

**Comment by klwong126** [comment]
Great! Is that code available for use?

**Comment by Niu-zhi-yi** [comment]
Hello, can you publicly use this model for real-time testing code?
